github/messages_test.go

```diff
@@ -82,6 +83,54 @@ func TestValidatePayload(t *testing.T) {
 	}
 }
 
+func TestValidateFormPayloadGET(t *testing.T) {
```

Comment:
That's a good call. It's better to add a separate test, then we can be more confident the old one still passes without changes.